### PR TITLE
Add 111th CatchUp summary

### PR DESCRIPTION
**Pull request by HarshKapadia2**
Please review and merge the 111th OTC CatchUp summary. Sorry for the delay.

### CODE REVIEW

File: summary/sessions/111/content.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Date: 24-12-2022
+
+Duration: 5 hrs 32 mins
+
+==== Topics Discussed
+
+* We held our first link:https://talks.ourtech.community[OTC Talks^] session with link:https://twitter.com/rishit_dagli[Rishit Dagli^] giving a talk 'Supercharge your deployments with WASM'.
```
Comment (tusharnankani) [inline summary/sessions/111/content.adoc:7]:
```suggestion
* We held our first link:https://talks.ourtech.community[OTC Talks^] session with link:https://twitter.com/rishit_dagli[Rishit Dagli^] giving a talk on 'Supercharge your deployments with WASM'.
```

**Review by tusharnankani — COMMENTED** [review]
LGTM!

File: summary/sessions/111/content.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ** The excellent talk led to an excellent discussion on Operating Systems (OSs), Processors, Virtual Machines (VMs) and hardware in general. Some of the things we talked about are listed in points below.
+* link:https://www.linkedin.com/in/anilharwani[Anil Harwani^] talked about how link:https://en.wikipedia.org/wiki/Ahead-of-time_compilation[Ahead-of-Time (AoT) compilation^] can have security implications on the client, because although storage and network is monitored, compute access is allowed, so Cryptocurrency mining can be done.
```
Comment (SirusCodes) [inline summary/sessions/111/content.adoc:9]: As far as I remember Anil doesn't wants his links in summary 👀

**Review by SirusCodes — CHANGES_REQUESTED** [review]
Rest looks good to me

File: summary/sessions/111/content.adoc (hunk 2 above)
Comment (HarshKapadia2) [reply summary/sessions/111/content.adoc:9]: Are you talking about linking his LinkedIn profile?

Comment (SirusCodes) [reply summary/sessions/111/content.adoc:9]: Yup

Comment (HarshKapadia2) [reply summary/sessions/111/content.adoc:9]: Okay, I'll remove his links. I've messaged him again though.

**Comment by HarshKapadia2** [comment]
@SirusCodes please review and merge.

**Review by SirusCodes — APPROVED** [review]

**Pull request merged**